app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb

```diff
@@ -225,6 +230,44 @@ def process_host_hardware_disks(extra_attributes)
         end
       end
 
+      def get_object_store
+        return unless @storage_service_name == :swift
+        @os_handle.service_for_each_accessible_tenant('Storage') do |svc, t|
+          svc.directories.each do |fd|
+            result = process_collection_item(fd, :cloud_object_store_containers) { |c| parse_container(c, t) }
+            process_collection(fd.files, :cloud_object_store_objects) { |o| parse_object(o, result, t) }
```

Comment:
There are a lot of single and double letter block vars here.  I can guess at `fd` and `o`.  But, I have no idea about `t`.

* `fd` = Directory (prefixed with "f" for some reason), or maybe this is "file descriptor"?
* `o` = Object, but the API call is `files`, which makes me think this is *really* a file, so maybe should be called `file`
* `t` = ???

Can you at least rename `t` to something more descriptive here?  Also, I think `fd` should probably be `dir`.  And, consider changing `o` to `file`.